devtools/cli/src/main/java/io/quarkus/cli/Create.java

```diff
@@ -21,16 +21,20 @@
 
     @CommandLine.Option(names = { "-a",
             "--artifact-id" }, order = 2, paramLabel = "ARTIFACT-ID", description = "The artifactId for project")
-    String artifactId = "my-project";
+    String artifactId = "code-with-quarkus";
 
     @CommandLine.Option(names = { "-v",
             "--version" }, order = 3, paramLabel = "VERSION", description = "The version for project")
-    String version = "1.0-SNAPSHOT";
+    String version = "1.0.0-SNAPSHOT";
 
     @CommandLine.Option(names = { "-0",
             "--no-examples" }, order = 4, description = "Generate without example code.")
     boolean noExamples = false;
 
+    @CommandLine.Option(names = { "-e",
+            "--examples" }, order = 4, description = "Choose which example(s) you want in the generated Quarkus application.")
+    Set<String> examples;
```

Comment:
Codestarts are now called examples? This is a bit confusing

Reply:
Well, for a non dev user there is no such notion of codestarts in code.quarkus.io or in any of the tooling. Also the existing option was called `noExamples`.

Also the kind of codestarts we are talking here are codestarts of type `code` and tagged `example`. 

By default, this is dealt thanks to the extension codestart mapping, but this option let you override the default to choose your own set of examples.

Reply:
cc @maxandersen

Reply:
@gastaldi 👆